### PR TITLE
fix some things in typings

### PR DESCRIPTION
**Pull request by na-2n**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -570,9 +570,9 @@ declare module 'eris' {
   }
 
   export class Collection<T extends { id: string }> extends Map<string, T> {
-    baseObject: new () => T;
+    baseObject: new (...args: any[]) => T;
     limit?: number;
-    constructor(baseObject: new () => T, limit?: number);
+    constructor(baseObject: new (...args: any[]) => T, limit?: number);
```
Comment (aurieh) [inline index.d.ts:575]: The typings use `Array<any>` in most places, you should keep this in mind

Comment (na-2n) [reply index.d.ts:575]: using `Array` for simple types is meme tho

Comment (aurieh) [reply index.d.ts:575]: Then make a PR changing all array types I used to `type[]` 🙃

Comment (na-2n) [reply index.d.ts:575]: me too thanks

Comment (cryptiklemur) [reply index.d.ts:575]: `type[]` is standard for simple types in tslint

Comment (aurieh) [reply index.d.ts:575]: @aequasi I'm fully aware of that, couldn't break my habit of using Array<T> before. I might PR the change at some point, unless someone else wants to do it. Before that, however, I would like the typings' style to stay consistent.

**Comment by na-2n** [comment]
closing cuz aurieh added my changes to his pull

**Pull request closed without merging**